src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    accuracy = 0.001
</pre><p>Why being so loose on the tolerance ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Changed it so that it gives the same results as in the other notebooks

Reply:
Changed in dfe233f7